EIPS/eip-1895.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 1895
+title: Support for an Elliptic Curve Cycles
+author: Alexandre Belling <alexandrebelling8@gmail.com>
+discussions-to: https://ethresear.ch/t/reducing-the-verification-cost-of-a-snark-through-hierarchical-aggregation/5128
+status: Draft
+type: Standards Track
+category: Core
+created: 2018-31-03
+---
+
+# Support for an Elliptic Curve Cycles
```

Comment:
The title doesn't need to be here. During parsing the title from the header is used.